packages/schematics/angular/migrations/migration-collection.json

```diff
@@ -19,6 +19,11 @@
       "version": "7.1.0-beta.0",
       "factory": "./update-7/index#typeScriptHelpersRule",
       "description": "Update an Angular CLI project to version 7."
+    },
+    "migration-05": {
+      "version": "7.0.3",
```

Comment:
Yesterday we were talking about not adding migrations on minors/patch versions, so I'm not sure if we should add this one.

Reply:
In that case maybe I should add it as `7.1.0-beta.0`, what do you reckon?

Reply:
I think this is really fixing a deficiency in the 7.0 update process.  So it would make sense to have it in a 7.0 patch release so that anyone updating from earlier majors will immediately benefit.

Reply:
So the logic is that newly updated projects (6->7) will include this migration, but that we don't expect users to run update from 7->7 and to get it, right? I'm ok with that.